model.js

```diff
@@ -16,11 +21,15 @@ module.exports = function() {
       apartments.ttl = ttl;
       apartments.metadata = {
         name: `${city} ${type}`,
-        description: `Craigslist ${type} listings proxied by https://github.com/dmfenton/koop-provider-craigslist`
+        description: `Craigslist ${type} listings proxied by https://github.com/dmfenton/koop-provider-craigslist`,
+        hasStaticData: false,
```

Comment:
I think `hasStaticData` is a field name we should change to something more meaningful like we do with others in FeatureServer